input/blog/2020-12-10-cake-vscode-extension-release-1.0.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cake VSCode Extension Release 1.0.0
+category: News
```

Comment:
I would use `Release Notes` category and use `News` just for not release related news (e.g. like joining DNF)

```suggestion
category: Release Notes
```